reference/docs-conceptual/community/contributing/get-started-writing.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Get started contributing to PowerShell documentation
+description: This article is an overview of how to get started as a contributor to the PowerShell documentation.
+ms.date: 03/05/2020
+ms.topic: conceptual
+---
+# Get started contributing to PowerShell documentation
+
+## PowerShell-Docs structure
+
+The [PowerShell-Docs repository][psdocs] is divided into two groups of content. Git branches are
+used to manage how and when documentation gets published.
+
+### Reference content
+
+The reference content is the PowerShell cmdlet reference for the cmdlets that ship in PowerShell.
+The [reference][ref] is collected in versions folders (5.1, 6, 7.0, and 7.1). This content contains
+cmdlet reference only for the modules that ship with PowerShell. This content is also used to create
+the help information displayed by the `Get-Help` cmdlet.
```

Comment:
Should we have a link here to point to where to contribute to other ref content, such as Azure PowerShell?

Reply:
meh